packages/components/src/Combobox/components/ComboboxContent/ComboboxHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Typography } from "@jobber/components/Typography";
+import { Button } from "@jobber/components/Button";
+import styles from "./ComboboxHeader.css";
+
+interface ComboboxHeaderProps {
+  subjectNoun?: string;
+  selectedCount: number;
+}
+
+export function ComboboxHeader(props: ComboboxHeaderProps): JSX.Element {
+  const label = props.subjectNoun ? `Select ${props.subjectNoun}` : "Select";
+  const actionText = props.selectedCount > 0 ? "Clear" : "Select all";
+
+  return (
+    <div className={styles.header} data-testid="ATL-Combobox-Header">
+      <Typography textColor="heading" fontWeight="semiBold">
+        {props.selectedCount === 1 ? `${props.selectedCount} selected` : label}
```

Comment:
assuming this is also temp

once it's real, we can probably make a single var of `const hasSelected = props.selectedCount > 0`

and then use that for the actionText and this bit of logic to show the `N selected` or the label

Reply:
Exactly, also temp.  Okay I love that and will simplify this nicely